conf/drupal/config/metatag.metatag_defaults.global.yml

```diff
@@ -8,11 +8,11 @@ id: global
 label: Global
 tags:
   canonical_url: '[current-page:url]'
-  image_src: 'https://www.midcamp.org/themes/custom/hatter/o-midcamp.png'
+  image_src: 'https://www.midcamp.org/sites/default/files/2019-10/MidCamp2020Social.png'
   title: '[current-page:title] | [site:name]'
-  og_image: 'https://www.midcamp.org/themes/custom/hatter/o-midcamp.png'
+  og_image: 'https://www.midcamp.org/sites/default/files/2019-10/MidCamp2020Social.png'
```

Comment:
Is this PR just waiting on an image to replace here @froboy ?

Reply:
Yup. Will update once #294 is deployed.

Reply:
@Brian-Clement this should be good to go how. I've uploaded the image using the new media and it just so happened to get the same url. :shipit: